### PR TITLE
[UCSC-131] Standardize card elements

### PR DESCRIPTION
**Pull request by bjcooper**
## What does this do/fix?

This is a normalization pass to standardize card anatomy and behaviors across the various card block patterns and template parts:

- Cards may have an image, overline, title, body, and CTA
- The font size of each card element should be consistent across variations
- The spacing between card elements should be consistent across variations
- A card itself is not clickable and should have no suggestive hover or focus interactions
- Linked images, overlines, titles, anchors in the body, and CTAs may be clickable
  - When the image is linked, it should exhibit a subtle zoom effect when interacted with
  - Non-linked overlines and titles should be black; linked overlines and titles should be light blue and turn dark blue with a yellow underline on hover
  - Links in the body should be styled as normal site links
  - CTAs should use their specified preset button colors

## QA

Links to relevant issues
- [UCSC-131](https://example.com)](https://moderntribe.atlassian.net/browse/UCSC-131)

Screenshots/video:
- https://www.loom.com/share/7c1de38da4034e6aa01164f59e9b01d2?sid=2d058cc6-d0b1-49b2-869b-995801ed7527

### CODE REVIEW

**Comment by bjcooper** [comment]
_bump_ @Vinsanity

File: src/scss/block-patterns/_cards.scss [hunk 1]
```diff
@@ -6,38 +8,71 @@
...
+/* CASE: Card title follows an overline */
+:where(.ucsc__card .has-small-font-size) {
+
+	+ .wp-block-heading,
+	+ .wp-block-post-title {
+		margin-block-start: -16px !important;
+	}
 }
 
+/* Card CTAs */
```
Comment (Vinsanity) [inline src/scss/block-patterns/_cards.scss:75]: let's remove this if it's not in use.

**Review by Vinsanity — APPROVED** [review]
Looking good Brett. Small change request and this should be ok to move to QA.

File: src/scss/block-patterns/_cards.scss (hunk 1 above)
Comment (bjcooper) [reply src/scss/block-patterns/_cards.scss:75]: There's one little CASE inside there that sets some special CTA styles when inside of a card grid, so it is in use.

**Pull request merged**